include/odp/api/abi-default/comp.h

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2017-2018, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+#ifndef ODP_ABI_COMP_H_
+#define ODP_ABI_COMP_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
+
+/** @internal Dummy type for strong typing */
+typedef struct { char dummy; /**< @internal Dummy */ } _odp_abi_comp_session_t;
+
+/** @ingroup odp_comp
```

Comment:
This group name needs to match the name defined with defgroup tag in the spec file (odp_compression).

Reply:
Has this comment by @psavol been addressed?

Reply:
Done